src/validation/input-params.ts

```diff
@@ -37,6 +37,18 @@ type TypeFromTypeString<T extends ParameterType> = T extends 'string'
   ? TypeFromDefinition<T>
   : never
 
+/**
+ * All posible types for specifying options for an InputParameter
+ */
+type ParameterOptions = string[] | readonly string[] | number | readonly number[]
```

Comment:
Should be this right?
```suggestion
type ParameterOptions = string[] | readonly string[] | number[] | readonly number[]
```

Reply:
Fixed